src/main/groovy/com/vantiq/gradledcomposeplugin/DcomposePlugin.groovy

```diff
@@ -62,7 +62,7 @@ class DcomposePlugin implements Plugin<Project> {
     public static final String TASK_GROUP_SERVICE_TEMPLATE = "$TASK_GROUP '%s' service"
     public static final String CONFIGURATION_NAME = "dcompose"
     public static final String EXTENSION_NAME = "dcompose"
-    public static final String DOCKER_DEPENDENCY = 'com.github.docker-java:docker-java:3.2.5'
+    public static final String DOCKER_DEPENDENCY = 'com.github.docker-java:docker-java:3.4.0'
```

Comment:
Not sure why dependencies are managed this way, but for now I'm not changing it.